samples/extensions/shader_object/shader_object.cpp

```diff
@@ -230,6 +230,11 @@ void ShaderObject::setup_framebuffer()
 // Create render pass for UI drawing
 void ShaderObject::setup_render_pass()
 {
+	// delete existing render pass
+	if (render_pass != VK_NULL_HANDLE)
```

Comment:
Hi @asuessenbach , per discussion today, can  you open an issue so we don't lose the context?  We're going to merge this one soon w/o capturing this - but want to circle back later

Reply:
Sorry, I missed the meeting where this was discussed...
Why should this issue not be resolved right here?

Reply:
The idea is to fix this after this PR has been merged. The PR is kinda urgent, and the line you highlighted is not new to the PR but already present in the base sample.

Reply:
@SaschaWillems so it seems, you know about the issue to be resolved later. Would you be so kind and file that issue, instead of me guessing what's meant to be done here?

Reply:
Sorry, I was under the impression that this change was already present beforehand. But as this code is new to this PR you're right to mention this and it indeed needs fixing in this PR.

Reply:
I did this change because renderpass was being created by the vulkansample in the framework automatically.  I can take this change out, but then validation layers correctly identifies a render_pass object not getting released.  The rest of the changes are updated to conform with your requests.

Reply:
That makes sense. We should keep that change then :)